src/spork.h

```diff
@@ -75,15 +88,41 @@ class CSporkMessage
         }
     }
 
+    /**
+     * GetHash returns the double-sha256 hash of the serialized spork message. The
```

Comment:
`The signature field is not included in the serialization.` this will not be true after #2288